### ISSUE TITLE
Request for Fixed Y scale on serial plotter

### ISSUE THREAD

**Issue by Bort-Plate** [arduino/Arduino#4527]
With values that constantly change, the range of the plot will shrink and expand, making it incredibly difficult to judge the amplitude of a plot without constantly re-checking the Y-axis (of which is too small to read on my 4K screen anyway).

By having a selectable minimum and maximum, each peak on the plot can be visually compared properly.

**Comment by cousteaulecommandant** [comment]
I agree that this is annoying; if not a selectable range (which I think would be best), at least an option to disable auto-scaling.

Also a horizontal grid would help.

**Comment by karlTH** [comment]
serial plotter is an awesone tools
a csv export will be really good.
thanks a lot for the work.
i m also agree with cousteaulecommandant

**Comment by andreashove** [comment]
A quick fix to this is to print your desired scale every time you print a value. 

If your values are between 0 and 1000 you can do:

```
String withScale = "0 ";
withScale += value;
withScale += " 1000";

Serial.println(withScale);
```

**Comment by MickMad** [comment]
I totally agree. Autorange is no good for me. I am currently sending a value between 0.0 and 1.0 to the serial plotter and it autoranges to -6.0 to +6.0 ... Why?

@andreashove we should not include a lot of overhead in memory and time of execution to our codes because a basic feature is missing.

**Comment by phoenixperry** [comment]
I am going to agree that this is really difficult to work with and could use a fix.

**Comment by alecokas** [comment]
I agree with @MickMad. Has any progress been made on this issue? I think it would be a useful addition.

**Comment by steeley** [comment]
yep, 3 years on and still no option....
Plotter is very useful, but on/off autoscale should have been a fundamental feature.
I had to hack  together my own plotter as part of a desktop app, but would be easier
in Arduino IDE.
